airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSource.java

```diff
@@ -236,18 +234,20 @@ private static AirbyteStream removeIncrementalWithoutPk(final AirbyteStream stre
 
   @Override
   public Set<JdbcPrivilegeDto> getPrivilegesTableForCurrentUser(JdbcDatabase database, String schema) throws SQLException {
-    return database.bufferedResultSetQuery(
-        conn -> conn.getMetaData().getTablePrivileges(getCatalog(database), schema, null),
-        resultSet -> JdbcPrivilegeDto.builder()
-            .grantee(ofNullable(resultSet.getString(GRANTEE)).orElse(EMPTY))
-            // we need the schema as different schemas could have tables with the same names
-            .schemaName(ofNullable(resultSet.getString(JDBC_COLUMN_SCHEMA_NAME)).orElse(EMPTY))
-            .tableName(ofNullable(resultSet.getString(JDBC_COLUMN_TABLE_NAME)).orElse(EMPTY))
-            .privilege(ofNullable(resultSet.getString(PRIVILEGE)).orElse(EMPTY))
```

Comment:
It seems that  
protected static final String GRANTEE = "GRANTEE";
protected static final String PRIVILEGE = "PRIVILEGE";

in AbstractJdbcSource class
will not be used after this change, if so could you pls remove them.